tests/conftest.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+
+@pytest.fixture
+def my_tmp_path(tmp_path):
+    return tmp_path
+
+
+@pytest.fixture
+def construct_wrong_fake_configs(my_tmp_path):
+    wrong_fake_configs = {
+        "paths": {
+            "ymldb": my_tmp_path / "data/ymldb",
```

Comment:
Here and everywhere else: the `/` operator of paths is there also for portability, since it works on all supported systems, but the character `"/"` does not.
So, specify each piece of the path on its own:
```suggestion
            "ymldb": my_tmp_path / "data" / "ymldb",
```